pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "phenopy"
+version = "0.5.3"
+description = "Phenotype comparison scoring by semantic similarity."
+authors = [
+    "Kevin Arvai <karvai@genedx.com>",
+    "Kyle Retterer <retterer@genedx.com>",
+    "Carlos Borroto <cborroto@genedx.com>",
+    "Vlad Gainullin <vgainullin@genedx.com>",
+    "Vincent Ustach <vustach@genedx.com>"
+]
+readme = "README.md"
+license = ""
+
+[tool.poetry.scripts]
+phenopy = "phenopy.__main__:main"
+
+[tool.poetry.urls]
+homepage = "https://github.com/GeneDx/phenopy"
+"Bug Tracker" = "https://github.com/GeneDx/phenopy/issues"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+fire = "*"
```

Comment:
Generally we should probably avoid "*" for all but the most stable libraries.
I imaging that just `poetry add`ing each of these would result in sensible pinnings.